localization/js/language-button.js

```diff
@@ -0,0 +1,21 @@
+import { Component, Property } from '@wonderlandengine/api';
+import { CursorTarget } from '@wonderlandengine/components';
+
+/**
+ * language-button
+ */
+export class LanguageButton extends Component {
+    static TypeName = 'language-button';
+    /* Properties that are configurable in the editor */
+    static Properties = {
+        langCode: Property.string('en')
+    };
+
+    init() {
+        this.object.getComponent(CursorTarget).onClick.add(this.onClick.bind(this));
```

Comment:
The `onClick` will leak when this component is destroyed, you should either:
* Add the listener in `onActivate` and remove it in `onDeactivate`
* Remove the listener in `onDestroy`

You will need to store the reference to `this.onClick.bind(this)`.

BTW @Squareys please double check you also clean listeners properly in the components in general.